javascript/javascript3/week2/javascript3-homwork-week2/Find  Cool Api/main.js

```diff
@@ -0,0 +1,38 @@
+fetch("computers.json")
+  .then((response) => response.json())
+  .then((data) => {
+    const computerList = document.querySelector("#computer-list");
+
+    data.computers.forEach((computer) => {
+      const computerCard = document.createElement("div");
+      computerCard.className = "computer-card";
+
+      const heading = document.createElement("h2");
+      heading.textContent = `${computer.brand} ${computer.model}`;
+
+      const screenSize = document.createElement("p");
+      screenSize.textContent = `Screen Size: ${computer.screen_size}`;
+
+      const processor = document.createElement("p");
+      processor.textContent = `Processor: ${computer.processor}`;
```

Comment:
This could be significantly shortened with this function:

```javascript
function createAndAppendElement(parent, elementType, elementText) {
  // Create a new element of the specified type
  const newElement = document.createElement(elementType);

  // Set the text content of the new element
  newElement.textContent = elementText;

  // Append the new element to the parent element
  parent.appendChild(newElement);
}
```

Reply:
Yes, you right!👍